common/model-views.cpp

```diff
@@ -523,7 +523,7 @@ namespace rs2
         bool* options_invalidated,
         std::string& error_message)
     {
-        option_model option;
+        option_model option = {};
```

Comment:
Why is it better than before?

Reply:
According to Ohad research looks like this zeros all native members